openmetadata-ui/src/main/resources/ui/playwright/utils/alert.ts

```diff
@@ -815,22 +819,26 @@ export const waitForRecentEventsToFinishExecution = async (
   await expect
     .poll(
       async () => {
-        const response = await apiContext
-          .get(
-            `/api/v1/events/subscriptions/name/${name}/eventsRecord?listCountOnly=true`
-          )
-          .then((res) => res.json());
-
-        return (
-          response.pendingEventsCount === 0 &&
-          response.totalEventsCount === totalEventsCount
-        );
+        try {
+          const response = await apiContext
+            .get(
+              `/api/v1/events/subscriptions/name/${name}/eventsRecord?listCountOnly=true`
+            )
+            .then((res) => res.json());
+
+          return (
+            response.pendingEventsCount === 0 &&
+            response.totalEventsCount === totalEventsCount
+          );
+        } catch (error) {
+          return false;
+        }
       },
```

Comment:
Why are we adding a try-catch block here? It won't cause the test to fail, even if a genuine error occurs.